lib/mayaUsd/undo/UsdUndoBlock.cpp

```diff
@@ -83,7 +83,7 @@ void UsdUndoBlockCmd::execute(const UsdUndoableItem& undoableItem)
     argUndoItem = UsdUndoableItem();
 }
 
-UsdUndoBlockCmd::UsdUndoBlockCmd(const UsdUndoableItem& undoableItem)
+UsdUndoBlockCmd::UsdUndoBlockCmd(UsdUndoableItem undoableItem)
     : _undoItem(undoableItem)
```

Comment:
Aren't we taking two copies here if we're not std::move'ing into _undoItem, one for the argument temporary, and one for the data member?  Perhaps we should be passing by rvalue reference?
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f18-for-will-move-from-parameters-pass-by-x-and-stdmove-the-parameter
Maybe these are things we can tweak later, and should not stop us from merging this branch.

Reply:
Agreed Pierre. There are many variations and honestly not a huge deal here.
https://www.nextptr.com/tutorial/ta1211389378/beware-of-using-stdmove-on-a-const-lvalue

Reply:
Indeed, let's be explicit about the intent of move and change line 87 to (I would hope compiler can deduce it, but well)
```
_undoItem(std::move(undoableItem));
```

This way at worst we get only one copy in case:
```
UsdUndoableItem myItems;
UsdUndoBlockCmd myCmd(myItems);
```
and zero-copy in case of
```
UsdUndoableItem myItems;
UsdUndoBlockCmd myCmd(std::move(myItems));
```